apps/todo/src/components/Button.tsx

```diff
@@ -0,0 +1,20 @@
+import { styled } from '@stitches/react';
+
+const StyledButton = styled('button', {
+  backgroundColor: '#E9D540',
+  borderRadius: '8px',
+  color: '#2F3740',
+  padding: '8px 46px',
+  border: 'none',
+  fontSize: '24px',
+  fontWeight: 'light',
+  fontFamily: 'Montserrat',
+  alignSelf: 'center',
+  marginTop: '24px',
+});
+
+const Button = ({ text }: { text: string }) => {
```

Comment:
Maybe something like 
```
const Button: React.FC<PropsWithChildren> = ({ children }) => {
  return <StyledButton>{children}</StyledButton>;
};
```
is better because you can then pass a `<span>` a `<div>` or whatever to the button instead of just text.